app/views/layouts/_top_bar.html.erb

```diff
@@ -99,5 +99,60 @@
         xmlhttp.send();
       }
     </script>
-  <% end %>
-</header>
+  </header>
+<% else %>
+  <header class="top-bar" id="top-bar" role="banner">
+    <a href="#articles-list" class="skip-content-link">Skip to content</a>
+    <div class="top-bar__container">
+      <a href="/" class="top-bar--logo top-bar--logo--logged-out" aria-label="<%= community_name %> Home">
+        <%= logo_svg %>
+      </a>
+
+      <div id="pwa-nav-buttons" class="pwa-nav-buttons">
+        <button type="button" id="app-back-button">
+          <%= inline_svg_tag("keyboard-left-arrow-button.svg", aria: true, width: 16, height: 16, class: "crayons-icon", title: "Back") %>
+        </button>
+        <button type="button" id="app-forward-button">
+          <%= inline_svg_tag("keyboard-right-arrow-button.svg", aria: true, width: 16, height: 16, class: "crayons-icon", title: "Forward") %>
+        </button>
+        <button type="button" id="app-refresh-button">
+          <%= inline_svg_tag("refresh-button.svg", aria: true, width: 16, height: 16, class: "crayons-icon", title: "Refresh") %>
+        </button>
+      </div>
+
+      <div class="top-bar--search top-bar--search--logged-out" id="top-bar--search">
+        <form acceptCharset="UTF-8" method="get" action="/search">
+          <input class="top-bar--search-input crayons-textfield" type="text" name="q" id="nav-search" placeholder="Search..." autoComplete="off" />
+        </form>
+      </div>
+
+      <div class="top-bar__links top-bar__links--logged-out">
+        <a href="/enter" class="crayons-btn crayons-btn--ghost-brand hidden mr-2 whitespace-nowrap s:block">
+          Log in
+        </a>
+        <a href="/enter" class="crayons-btn hidden mr-2 whitespace-nowrap s:block">
+          Create account
+        </a>
+
+        <div class="top-bar__menu s:hidden" id="top-bar__menu">
+          <a id="moderation-link" class="hidden top-bar__link trusted-visible-block" aria-label="Moderation" href="<%= mod_path %>">
+            <%= inline_svg_tag("mod.svg", aria: true, class: "crayons-icon", title: "Moderation") %>
+          </a>
+
+          <a href="/connect" id="connect-link" class="top-bar__link hidden" aria-label="Connect">
+            <%= inline_svg_tag("connect.svg", aria: true, class: "crayons-icon", title: "Connect") %>
+            <span class="crayons-indicator crayons-indicator--accent hidden" id="connect-number"></span>
+          </a>
+          <a href="/notifications" id="notifications-link" class="top-bar__link hidden" aria-label="Notifications">
+            <%= inline_svg_tag("bell.svg", aria: true, class: "crayons-icon", title: "Notifications") %>
+            <span class="crayons-indicator crayons-indicator--critical hidden" id="notifications-number"></span>
+          </a>
```

Comment:
For logged-out user, I initially removed this entire codeblock, but removing it didn't make the `top-bar__menu__trigger` open because of the way these interactions are set-up in `initializeSpecialNavigationFunctionality.js`. If anyone can help with refactoring this (either in this PR or a separate one), that would be much appreciated.